dist/web3.js

```diff
@@ -1444,6 +1444,9 @@ web3.setProvider = function (provider) {
     this.currentProvider = provider;
     RequestManager.getInstance().setProvider(provider);
 };
+web3.isConnected = function(){
```

Comment:
I still do not like this method here. It's not reliable and it gives you nothing. You still have to take care of random unpredicted errors (eg. geth node going down) in your code.